addon/components/nrg-dropdown.js

```diff
@@ -178,6 +167,16 @@ export default class NrgDropdownComponent extends NrgValidationComponent {
     );
   }
 
+  getDefaultValue() {
+    if (this.args.defaultValue !== undefined) {
+      return this.args.defaultValue;
+    }
+    if (this.args.multiple) {
+      return [];
```

Comment:
Before the constructor was setting the value to an ember array.
```js
if (this.args.multiple && !this.value) {
  this.value = A();
}
```
Could changing this to a regular array create a problem?

Reply:
Good catch - it probably will, so let me fix that